packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md

```diff
@@ -4,18 +4,29 @@ In the Admin Console, complete these steps:
 
 2. Click **Add Application**.
 
-3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code. Pick **Web** if your application is using the tokens solely in server-side code.
+3. From the **Add Application** page, click **Create New App**.
 
-4. Enter the **Name** of your app.
+4. Select a **Platform**:
+    - **Single Page App(SPA)**&mdash;for applications using tokens in Angular, React, Vue, or other browser-side code
```

Comment:
```suggestion
    - **Single Page App (SPA)**&mdash;for applications using tokens in Angular, React, Vue, or other browser-side code
```

Reply:
Thanks Brian. I've reorganized this section and added the space.